### PR TITLE
slate update, figure group

### PR DESCRIPTION
**Pull request by lukasbuenger**
Depends on orbiting/styleguide#63

New editor module figuregroup.

### CODE REVIEW

File: components/editor/modules/figuregroup/index.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  const figureSerializer = figureModule.helpers.serializer
+
+  const data = getData(node.data)
+
+  const result = {
+    kind: 'block',
+    type: TYPE,
+    data,
+    nodes: node.children.map(
+      v => figureSerializer.fromMdast(v)
```
Comment (tpreusse) [inline components/editor/modules/figuregroup/index.js]: caption serializer missing

File: components/editor/modules/figuregroup/index.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  parent,
+  {
+    visitChildren,
+    context
+  }
+) => {
+  const [ figureModule ] = subModules
+
+  const mdastChildren = node.nodes.map(v =>
+    figureModule.helpers.serializer.toMdast(v)
```
Comment (tpreusse) [inline components/editor/modules/figuregroup/index.js]: caption serializer missing

**Comment by lukasbuenger** [comment]
Changelog:
- new figurecaption module that is independent of owner (figure or figuregroup)
- removed after type insertion because I saw no obvious way of how to configure the template schema without creating massive boilerplate
- center module enforces a paragraph as last child to compensate for the lack of after type.

**Comment by lukasbuenger** [comment]
Check obviously fails because PR depends on orbiting/styleguide#66

**Pull request merged**